labrad/util/__init__.py

```diff
@@ -370,7 +375,15 @@ def runServer(srv, run_reactor=True, stop_reactor=True):
 
     @inlineCallbacks
     def run(srv):
-        reactor.connectTCP(config['host'], int(config['port']), srv)
+        host = config['host']
+        port = int(config['port'])
+        if config['use-tls']:
+            tls_options = crypto.tls_options(host)
+            reactor.connectSSL(host, port, srv, tls_options)
+        else:
+            srv._start_tls = config['tls'].lower() != 'off'
```

Comment:
I've added a method `configure_tls` on the server class to pass in the required configuration info.